### DISCUSSION TITLE
alternative feature: Cur

### DISCUSSION THREAD

**Discussion by mirsella** [manna-harbour/miryoku#297] [Q&A] [answered]
hello !

i'm using miryoku zmk, and i undestand how to activate and use the Opp alternative feature. but i can't manage to do anything with the Cur.
is there a problem with it ? can someone explain exactly to use this feature so that i can try to reproduce ?

thanks !

**Comment by manna-harbour** [comment, marked as answer]
See https://github.com/manna-harbour/miryoku_zmk#no-current-layer-lock.

**Reply by mirsella** [reply]
oh well sorry about this useless question... thank you for being always this helpful for everyone asking newbie questions !

**Reply by manna-harbour** [reply]
No problem! If you're particularly interested in this feature you could try the PR mentioned in the linked issue.

**Reply by mirsella** [reply]
yes i saw it but i didn't thought it would be that easy to use ! sadly i already use caksoylar/zmk/feat/mouse-keys-3.2 which have merge conflict when trying with ecstatic-morse/zmk/molock.

thanks !
